visual_behavior/data_access/loading.py

```diff
@@ -346,6 +346,23 @@ def dff_traces(self):
             self._dff_traces = super().dff_traces
         return self._dff_traces
 
+    @property
+    def dff_dataframe(self):
```

Comment:
Calling this “dff_dataframe” is a bit confusion because the dff_traces attribute is itself a dataframe. Maybe this should be called “dff_traces_tidy”. Actually, I think it might be preferable to keep this out of the dataset object entirely, since the purpose of the object is to provide the basic data streams. I would be in favor of adding a function to data_access.reformatting that turns the dff_traces attribute into a tidy dataframe. It can be just one line of code to do it, but then it keeps the dataset object streamlined, and is more explicit in what it is (a reformatting of an existing dataframe rather than a second dataframe with redundant information).

Reply:
I moved the function to reformat.py. Now called `get_tidy_dff_traces` and takes dataset as an input.